client/python/tests/integration/test_typing_gen.py

```diff
@@ -0,0 +1,27 @@
+import os
```

Comment:
I don't really think you need an integration test here.  I think the unit test sufficies.

Reply:
I see why you did this.  You want the writing of a file done as an integration test.  I like your approach but I don't think we should include this as an integration test.  

I would propose a refactor where we generate the typing file in memory and then you test that.  The prod code can write this buffer to disk and that shouldn't be tested.  

This is a pattern I like to follow with interacting with external resources.  

This is overkill with the code that you have but it is a good practice to learn.  For example, you do not want to create a database in order to unit test code.  So you can build a simple mock and tests your logic and you trust the DB library to be tested enough.

Reply:
Sorry this is not 100% clear, do you want me to remove this file, or refactor it so that the file is only stored in memory and that is asserted?

Reply:
Both.  

1) Remove the integration test file.  
2) Add a unit test that generates the file in memory and asserts it matches what you expect.
3) Production code should generate the file and then you write that out to dist.

Reply:
Cool. On it :))